test/framework/deployments/externalservice/kubernetes.go

```diff
@@ -60,29 +60,18 @@ func (k *k8SDeployment) Deploy(cluster framework.Cluster) error {
 }
 
 func (k *k8SDeployment) Delete(cluster framework.Cluster) error {
-	err := k8s.KubectlDeleteE(cluster.GetTesting(),
+	_ = k8s.KubectlDeleteE(cluster.GetTesting(),
```

Comment:
why are we ignoring errors? Ignoring errors in E2E led to issues in the past (we weren't deleting Kuma in a proper way and we were ignoring it). I would avoid running tests anymore if there is a problem with the cleanup.

Reply:
Well, there is a problem ensuring that "all" External service pods are deleted before the next test starts. In the other tests we do create test apps in `kuma-test` namespace, and then we just delete that namespace making sure it completely disappeared cleaning up all associated resources before we continue. Now here I tried to delete the pods one by one and only deleting the namespace if there are 0 pods. In practice, the pods were staying available just enough so that the namespace for the external services was never deleted, which caused confusion in the follow up tests.

I probably can just remove the deleting of the resource and let the namespace be deleted upon the first call to Delete.

Reply:
Ok, I understand. I'm a little bit worried that we cannot get K8S to work for this simple scenario, but I don't want you to spend time on this. We can go back to it when we will actually need the ability to delete one external service deployment and not the other.